README.md

```diff
@@ -310,40 +348,113 @@ aws ec2 describe-instances --region eu-west-1
 ./start_aws_controller
 ```
 
-## Example
+## Examples
 
-In the directory `plz/examples/python` there is a minimal example showing
-how to do input and output with `plz`. You can also see the directory
-`plz/tests/end-to-end` for examples on how to use parameters and measures,
-which this documentation doesn't cover yet.
-
-*Note: if you want to run the example using the AWS instances, be aware that
+*Note: if you want to run the examples using the AWS instances, be aware that
 this has a cost. You can change the value of
 `"max_bid_price_in_dollars_per_hour": N` in `plz.config.json` to any value
-you like. The example takes around 5 minutes to run.
-The value in the provided file is 0.5 dollars/hour. See the following note as
-well.*
+you like. Examples takes around 5 minutes to run.
+The value in the provided configs range from 0.5 dollars/hour to 2 dollars/hour
+(for GPU-powered machines). See the following note as well.*
 
 *Note: unless you add `"instance_max_uptime_in_minutes": null,` to your
-`plz.config.json`, the instance terminates after 60 minutes.  That's on
-purpose, in case you're just trying the tool and something doesn't go well
-(like, there's a power cut). You can always use `plz list` and `plz kill`
-before leaving your computer, as to make sure that there no instances remaining.
-For maximum assurance, you can check your instances in the AWS console.*
+`plz.config.json`, all AWS instances you start terminate after 60 minutes.
+That's on purpose, in case you're just trying the tool and something doesn't
+go well (like, there's a power cut). You can always use `plz list` and
+`plz kill` before leaving your computer, as to make sure that there no
+instances remaining. For maximum assurance, you can check your instances in
+the AWS console.*
+
+
+### Pytorch
+
+In the directory `plz/examples/pytorch` there's a full-fledged example that
+shows how to do the classical training for character recognition using
+LeNets, using a subset of the well-known MNIST dataset. There's a simple
+example using only python, described in the section below.
+
+Anything related to `plz` is in `main.py`. In fact the most relevant lines
+are the following ones:
+
+```
+def get_from_plz_config(key: str, non_plz_value: T) -> T:
+    configuration_file = os.environ.get('CONFIGURATION_FILE', None)
+    if configuration_file is not None:
+        with open(configuration_file) as c:
+            config = json.load(c)
+        return config[key]
+    else:
+        return non_plz_value
+[...]
+    input_directory = get_from_plz_config(
+        'input_directory', os.path.join('..', 'data'))
+    output_directory = get_from_plz_config('output_directory', 'models')
+    parameters = get_from_plz_config('parameters', DEFAULT_PARAMETERS)
+    measures_directory = get_from_plz_config('measures_directory', 'measures')
+    summary_measures_path = get_from_plz_config(
+        'summary_measures_path',
+        os.path.join('measures', 'summary'))
```

Comment:
Not that it matters for the purposes of this, but at this point you have read the poor file 5 times.